app/(auth)/signin.js

```diff
@@ -0,0 +1,172 @@
+import { Link } from "expo-router";
+import {
+  getAuth,
+  sendPasswordResetEmail,
+  signInWithEmailAndPassword,
+} from "firebase/auth";
+import { useState } from "react";
+import {
+  Image,
+  Keyboard,
+  Pressable,
+  StyleSheet,
+  Text,
+  TextInput,
+  TouchableWithoutFeedback,
+  View,
+} from "react-native";
+import { KeyboardAwareScrollView } from "react-native-keyboard-aware-scroll-view";
+import { currentAuthContext } from "~/context/Auth";
+import { auth } from "~/firebaseConfig";
+
+const BUTTON_WIDTH = 150;
+const INPUT_WIDTH = 200;
+
+export default function SignIn() {
+  const { signIn } = currentAuthContext();
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+  async function handleSignIn() {
+    try {
+      const userCredential = await signInWithEmailAndPassword(
+        auth,
+        email,
+        password,
+      );
+      console.log(userCredential.user);
+      signIn();
```

Comment:
Is this signin call necessary? It doesn't do anything

Reply:
removed